tests/test_block_creation_mvp.py

```diff
@@ -225,3 +225,36 @@ def test_transaction_wrong_payload(self, transaction_template, data_set):
             )
         except TypeError as e:
             assert str(e) == data_set["error"]
+
+    @pytest.mark.parametrize(
+        "transaction_template", [pytest.param(TransactionTemplates.legacy_transaction, marks=pytest.mark.skip
+            (reason='https://github.com/ledgerwatch/erigon/issues/5214'))])
+    def test_replace_transaction(self, transaction_template):
+
+        initial_txn_hash = self.send_transaction(
+            transaction_template(self.web3,
+                                 self.receiver_address,
+                                 self.sender_private_key))
+        # TODO: maybe add a verification that transaction is in pool
+
+        initial_txn_nonce = self.web3.eth.get_transaction(initial_txn_hash)["nonce"]
+
+        raw_txn = {
+            "from": self.sender_address,
+            "gasPrice": self.web3.eth.gas_price,
+            "gas": 6000000,
```

Comment:
Thank you @anastasiyaig 
Good idea of replacing transaction.
However, we missed with param a little. Currently you vary `gas` value. But we need to decrease `gasPrice` value (like it says in the issue #5214 ) in order transaction is hang in tx_pool, and then with subsequent transaction with high enough gasPrice - to replace the transaction.

Reply:
okay,  i will fix it

Reply:
@Serhy i have updated the logic of the test , so hopefully now it uses correct parameter (`gasPrice`). I am stuck on debugging why the test is failing on the verification for transaction hash of new transaction (with higher `gasPrice`). The assertion error is raised that this transaction has not been found in transaction pool. Either the `check_transaction_is_in_pool` method is not correct, or this replacing transaction is not sent to transaction pool. Need some help here.